core/blockchain.go

```diff
@@ -98,7 +101,7 @@ const (
 	diffLayerPruneRecheckInterval   = 1 * time.Second // The interval to prune unverified diff layers
 	maxDiffQueueDist                = 2048            // Maximum allowed distance from the chain head to queue diffLayers
 	maxDiffLimit                    = 2048            // Maximum number of unique diff layers a peer may have responded
-	maxDiffForkDist                 = 11              // Maximum allowed backward distance from the chain head
+	maxDiffForkDist                 = 13              // Maximum allowed backward distance from the chain head
```

Comment:
why 13?

Reply:
0->1->2->3->4->5->6->7->8->9->10->11->12->13->14->15->16
0->1'->2'->3'
If block 1' and block 2' get 2/3+, votes, but the block 3' isn't received because of some reasons, the upper fork may begin to increase, when it increases to block 16, the block 1 is naturally justified, then validators can begin vote on the upper fork, at this time we should reject block 3', otherwise, the block 3' will make the block 1' finalized, and the later vote on the upper fork may finalize block 16, this is not expected. So when the upper fork grows to block 16, we should reject block 3', which means the maxUncleDist should be 12, because of previous wrong justified logic, we set it to 13 previously.

Because the value 12 or 13 of the value is a little bit weird, I suggest using the original 11, keep the same value in the engineering realization not changed, will you agree?

Reply:
keep 11 not changed